src/Android/Xamarin.Android/Samples/Layers/DisplayKml/DisplayKml.cs

```diff
@@ -49,7 +49,7 @@ protected override void OnCreate(Bundle bundle)
         private void Initialize()
         {
             // Set up the basemap.
-            _mySceneView.Scene = new Scene(Basemap.CreateImageryWithLabels());
```

Comment:
The `ArcGISImagery` layer is using a vector layer for labels, so that won't work in scenes.

I think ImageryStandard works without issues, but that is one of a very small handful that work. 

I'm not sure if we want to avoid use of `BasemapStyle` in scenes altogether, or just be careful to only use styles that are backed by raster layers.

Reply:
These changes have been reverted. Plan is to move these to Basemap style when support for vector is added.